Packs/CortexAttackSurfaceManagement/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,35 @@
+
+#### Playbooks
+
+##### Cortex ASM - ASM Alert
+Replaced earlier **Cortex ASM - ASM Alert** playbook and Readme files.
```

Comment:
what is changed in the playbook? What difference should users expect?

Reply:
@dorschw , the original playbook was created really just as an "example" and not sure it was actually used.  It focused on enrichment whereas now there are additional stages (triage, enrichment, decision, remediation, ticketing, validation, summarization).  Let me know if you want me to add more data in this release note.

Reply:
Please mention the additional stages (`The playbook now allows ...`)